.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
         if: "${{ fromJson(inputs.github).event_name == 'pull_request' }}"
         env:
           IMAGE_TAG: "${{ github.sha }}"
-          IMAGE_REGISTRY: "localhost:5000/workbench-images"
+          IMAGE_REGISTRY: "ghcr.io/${{ github.repository }}/workbench-images"
```

Comment:
We aren't pushing this, so we can use any registry we like; so let's keep it consistent.